scripts/jenkins/run-benchmarks.sh

```diff
@@ -28,6 +28,10 @@ function setUp() {
     then
         CORE_INDEX=7
         BASE_FREQ="1.9GHz"
+    elif [ "${CPU_MODEL}" == "12th Gen Intel(R) Core(TM) i5-12500 " ]
+    then
+        CORE_INDEX=11
```

Comment:
Based on the other types, it seems to count the effective threads rather than physical cores, but I am not sure